components/eamxx/src/physics/spa/spa_functions_impl.hpp

```diff
@@ -91,12 +92,14 @@ create_horiz_remapper (
 
   const int ncols_model = model_grid->get_num_global_dofs();
   std::shared_ptr<AbstractRemapper> remapper;
-  EKAT_REQUIRE_MSG (ncols_data<=ncols_model,
-      "Error! We do not allow to coarsen spa data to fit the model. We only allow\n"
-      "       spa data to be at the same or coarser resolution as the model.\n");
-  if (ncols_data==ncols_model) {
+  if (ncols_data==ncols_model
+      or
+      use_iop /*IOP class defines it's own remapper for file data*/) {
     remapper = std::make_shared<IdentityRemapper>(horiz_interp_tgt_grid,IdentityRemapper::SrcAliasTgt);
   } else {
```

Comment:
For IOP, force remaper to be identity. `IOPReader` will automatically read file data onto the remapper target grid.